book/website/ethical-research/data-hazards/dh-how-to-use.md

```diff
@@ -0,0 +1,113 @@
+ 
+As you are collecting and analyzing your data, you might want to iteratively think of the potential Data Hazards that exist in the information you are actively collecting. 
+If you have a project where data has already been collected in the past, you can still apply and think of what labels may be relevant to the dataset. 
+
+When you are reporting your results, it is recommended you also think of reporting mitigation strategies together with the labels; examples of how others have done this can be found [here](https://datahazards.com/contents/materials/examples.html). 
+This would then be helpful for people who see your outputs in the future. 
+They can be aware of potential risks as they proceed with the project, and continue to think of solutions to any issues related to the research topic.
+
+(er-datahazardshowtouse-precautions)=
+## **3. Reflecting** on Safety Precautions
+
+This chapter is a good place to read about {ref}`er-self-reflection`, where discussion on identity and positionality, power and privilege, and self-reflection prompts are provided. 
+Reflecting on where you stand in your work means you are more likely to see and acknowledge mitigation strategies which can apply to data hazards of your work. 
+Likewise, reflecting on the history of where your work has come from may shed light as to how and why some data hazards apply to your work and promote thinking on how to alleviate these later. 
+
+Are you using data? Then doing some reflection as suggested above could help you think of what Data Hazards labels you might encounter throughout your project, for example ["ranks of classifies people hazard"](https://datahazards.com/contents/hazards/ranks-classifies.html) or ["risk to privacy"](https://datahazards.com/contents/hazards/risk-to-privacy.html).
+
+A key part of the Data Hazards framework is to reflect on what to do differently.
+Thinking about mitigation strategies means that you suggest ways to prevent risks associated with your work, and if no prevention is possible at present, how to avoid it in the future. 
+The point of this is not to think of *all* the possible scenarios on how your work carries or can carry risks, and of *all* possible mitigation measures; but to reflect, acknowledge and promote an awareness about the ethical implications of the work that we do.  
+
+(er-datahazardshowtouse-display)=
+## **4. Displaying** the Data Hazard labels 
+
+To showcase your reflections and how you have applied the labels, displaying them can help people visualise their importance in your work.
+
+The Data Hazards website has some suggestions and templates on [how to present the labels](https://datahazards.com/contents/materials/presenting.html
+). 
```

Comment:
Not sure if this line break will break the link, but to be sure, put the dangling `)` on the same line here.